libraries/splunk_password.rb

```diff
@@ -8,7 +8,7 @@
 # Module contains different functions to encrypt and decrypt splunk passwords
 module CernerSplunk
   # Encrypts the password before writing into config files. As of now all the passwords
-  # needs to be XORed except for the sslKeyFilePassword. The boolean
+  # needs to be XORed except for the sslKeyFilePassword / sslPassword. The boolean
```

Comment:
minor, should be sslKeysfilePassword (the misspelling kept me from seeing this when I searched through the code). Or just remove it since that attribute is deprecated in splunk.